Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: ci-validate
+ci-validate: check-reference
+
+# Basic lint checking
+lintCheck:
```

Comment:
In 4.18 we call this from ci-validate, right? is there a reason the 4.17 does not?

Reply:
Correct.  In 4.17 we have the infrastructure to call it, but not the changes @imiller0  made to make the yaml files complaint.  So enabling this, without also backporting those changes, would make the CI fail.